### ISSUE TITLE
Frequency of toast notification in joystick onMove() method

### ISSUE THREAD

**Issue by livmari** [DIT112-V20/group-08#119]
### User story
As an end-user, I want to receive relevant toast notifications, so that I can tell which state of the application I am currently interacting with.

### Acceptance criteria / bug explanation
The toast notification in the onMove() method of the joystick in MainActivity is reposted every time the joystick is moved. The onMove() can happen a large number of times before the end-user lets go of the joystick. This means that the toast message prompting the user to connect the car before using the joystick is printed too many times (should only be once), and remains on screen for ages.

This should be considered a bug, even though the system it is technically doing exactly what we told it to.

If it can be fixed, that would be nice :)

**Comment by ZJW-92** [comment]
I agree. I don't know if it work that I put switch case inside if else statements. I mean, if (!carIsConnected) {
            pleaseConnectCar();
        } else { switch case 1, switch case 2.... }}

**Comment by livmari** [comment]
@ZJW-92 , it won't work because everything is still within a method that activates on move screen touch. So the toast will be displayed multiplied by the on move time.
